### ISSUE TITLE
Migrations 2.6.13 -> 2.7.6 fail

### ISSUE THREAD

**Issue by sjoerdie** [open-formulieren/open-forms#4680]
### Product versie / Product version

2.6.13

### Customer reference

Taiga Haarlemmermeer 168

### Omschrijf het probleem / Describe the bug

Migrations fail, sentry issue available in taiga ticket. 

### Screen resolution

None

### Device

None

### OS

None

**Comment by sergei-maertens** [comment]
<details>
<summary>Traceback</summary>

```
JSONDecodeError: Expecting value: line 1 column 1 (char 0)
  File "manage.py", line 24, in <module>
    execute_from_command_line(sys.argv)
  File "django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
  File "django/core/management/base.py", line 106, in wrapper
    res = handle_func(*args, **kwargs)
  File "django/core/management/commands/migrate.py", line 356, in handle
    post_migrate_state = executor.migrate(
  File "django/db/migrations/executor.py", line 135, in migrate
    state = self._migrate_all_forwards(
  File "django/db/migrations/executor.py", line 167, in _migrate_all_forwards
    state = self.apply_migration(
  File "django/db/migrations/executor.py", line 252, in apply_migration
    state = migration.apply(state, schema_editor)
  File "django/db/migrations/migration.py", line 132, in apply
    operation.database_forwards(
  File "django/db/migrations/operations/special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "openforms/forms/migrations/0097_v267_to_v270.py", line 19, in fix_forms_simple_conditionals
    config_modified = convert_simple_conditionals(definition.configuration)
  File "openforms/formio/migration_converters.py", line 213, in convert_simple_conditionals
    component["conditional"]["eq"] = json.loads(eq)
  File "__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
```

</details>

**Comment by sergei-maertens** [comment]
The fix will be applied to the stable/2.7.x branch only - in 2.8.x this migration no longer exists, and in 2.6.x it didn't exist yet.
